deployment.environment RUM value (#1414)
SumoLogic/sumologic-documentation: docs/apm/real-user-monitoring/dashboards.md

Once the RUM app has been installed, use our [Explore view](/docs/dashboards-new/explore-view.md) to gain visibility into your web app's performance and end-user activity, such as geographic location, browser type, operating systems used. These dashboards visualize RUM metrics gathered from browser tracing instrumentation.

1. Go to **+New** > **Explore**.
1. Under **Explore By**, **click Real User Monitoring**.

> 1. Select your desired dashboard from **Dashboard** dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)<br/> There are three dashboard types on the **Application** and **Service** level and a single one on the **Action type** and **Action** level.

Reviewer: There are three dashboard types on the **Application**, **Service** and **Environment** levels

Author: thank you!

> 1. Select your desired dashboard from **Dashboard** dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)<br/> There are three dashboard types on the **Application** and **Service** level and a single one on the **Action type** and **Action** level.

Author: ```suggestion
1. Select your desired dashboard from **Dashboard** dropdown menu in the header:<br/>![explore rum with red box.png](/img/rum/explore-rum-with-red-box.png)<br/> There are three dashboard types on the **Application**, **Service**, and **Environment** levels, and a single one on the **Action type** and **Action** levels.
```

     * **route changes**: single-page-app specific way to navigate to a new page/view without having to load a new document.
   * **Action Name**: automatically generated from URLs. No configuration is required. The specifics of it will depend on action type. Action names can contain asterisks (`*`) to replace automatically-detected dynamic parts of the URL. If you have action names that overlap, the action name with an asterisk contains data for page loads NOT contained in more specific action names. For example, `http://www.site.com/path/page.htm` does not contain actions from `http://www.site.com/path/*`.

Example:<br/><img src={useBaseUrl('img/rum/explore-view-rum.png')} alt="Real User Monitoring" />

## Navigating RUM Dashboards

### RUM Overview Dashboards

The RUM Overview dashboards show the user experience for performance and requests metrics for your selected application, service, and action. Data is broken down by top geo-locations, operating systems, and browsers. These dashboards include:
* **RUM Overview - Application**

> * **RUM Overview - Application Service**

Reviewer: + 
**RUM Overview - Application Service with Environment**

> * **RUM Overview - Application Service**

Author: ```suggestion
* **RUM Overview - Application Service**
* **RUM Overview - Application Service with Environment**
```

* **RUM Overview - Navigation Action**

> * **RUM Overview - Load Action**

Reviewer: I dont think it's there

Author: @pbrzoska-sumo , can you clarify what action I should take with those four items?

Reviewer: correct name is "RUM Overview - DocumentLoad actions"

Reviewer: The list is incomplete and ordered in a wrong way. So either fix or remove all together. It is confusing in its current form.

![RUM Overview](/img/rum/RUM-Overview.png)

![RUM-Overview-Application-Service](/img/rum/RUM-Overview-Application-Service.png)

### RUM TopN Application/Service

The **RUM - TopN - Application** and **RUM - TopN - Application Service** dashboards show top N browsers, operating systems, and geographical locations by load time and requests for the selected **application**, **service**, or **environment**.

Use these dashboards to:
* Find out top N browsers, operating systems, and geolocations by load or requests.
* Understand the slowest and fastest browsers from a rendering perspective or geographical locations from a network perspective.
* Understand XHR and log errors your users are experiencing.
* Find out which browsers and operating systems are in use by your users and where are they are geographically located.

You can select the timing metric type in the **statistic** dropdown on the dashboard header. This will change the browser time metrics types on charts. You can also define the top N number for all charts.

![img](/img/rum/RUM-TopN-Application.png)

### RUM Performance Analytics Application/Service

> The **RUM Performance Analytics - Application** and **RUM Performance Analytics - Application Service** dashboards show the page performance and requests for a cohort of users specified by selecting the desired combination of dimensions.

Reviewer: ```suggestion
The **RUM Performance Analytics** dashboards for **Application**, **Service** and **Environment** show the page performance and requests for a cohort of users specified by selecting the desired combination of dimensions.
```

Use these dashboards to:

> * Filter data for specific combinations of application, browser, operating system, and/or geolocation.

Reviewer: ```suggestion
* Filter data for specific combinations of browser, operating system, and/or geolocation.
```

   * **Action Type**: can be one of:
     * **document loads**: representing loading of actual documents and their resources into the browser
     * **XHR actions**: representing any interaction with a page like click or submit that executes AJAX requests in the background to communicate with the backend, or
     * **route changes**: single-page-app specific way to navigate to a new page/view without having to load a new document.
   * **Action Name**: automatically generated from URLs. No configuration is required. The specifics of it will depend on action type. Action names can contain asterisks (`*`) to replace automatically-detected dynamic parts of the URL. If you have action names that overlap, the action name with an asterisk contains data for page loads NOT contained in more specific action names. For example, `http://www.site.com/path/page.htm` does not contain actions from `http://www.site.com/path/*`.

Example:<br/><img src={useBaseUrl('img/rum/explore-view-rum.png')} alt="Real User Monitoring" />

## Navigating RUM Dashboards

### RUM Overview Dashboards

> The **RUM Overview** dashboards (**Application**, **Service**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers.

Reviewer: ```suggestion
The **RUM Overview** dashboards (**Application**, **Service**, **deployment.environment**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers.
```

> The **RUM Overview** dashboards (**Application**, **Service**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers.

Reviewer: ```suggestion
The **RUM Overview** dashboards (**Application**, **Service**, **Service with environment**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers.
```

> The **RUM Overview** dashboards (**Application**, **Service**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers. These dashboards include:

Author: ```suggestion
The **RUM Overview** dashboards (**Application**, **Service**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers.
```

Use these dashboards to:
* Analyze load and paint timings for page document loads by application, service, or action.
* View information about core web vitals, XHR processing times/errors, and log errors.
* Understand what top browsers, operating systems, and geolocations are active with your website.

You can select the timing metric type in the **statistic** dropdown on the dashboard header. This will change the browser time metrics types on charts.

You can also click on any data-point on the charts to open a details panel and view the **Infrastructure** tab to drill down to traces representing user transactions from the selected time point.

> Overview dashboards on all Explore levels have a panel showing geographical user activity for the selected entity. Geographic dashboards do not measure performance.

Reviewer: ```suggestion
Overview dashboards on all Explore levels have a panel showing geographical user activity for the selected entity. Geographic dashboards  measure user activity as dot size and performance as a color.
```

> Overview dashboards on all Explore levels have a panel showing geographical user activity for the selected entity. Geographic dashboards  measure user activity as dot size and performance as a color.

Author: ```suggestion
Overview dashboards on all Explore levels have a panel showing geographical user activity for the selected entity. Geographic dashboards measure user activity as dot size and performance as a color.
```

   * **Action Type**: can be one of:
     * **document loads**: representing loading of actual documents and their resources into the browser
     * **XHR actions**: representing any interaction with a page like click or submit that executes AJAX requests in the background to communicate with the backend, or
     * **route changes**: single-page-app specific way to navigate to a new page/view without having to load a new document.
   * **Action Name**: automatically generated from URLs. No configuration is required. The specifics of it will depend on action type. Action names can contain asterisks (`*`) to replace automatically-detected dynamic parts of the URL. If you have action names that overlap, the action name with an asterisk contains data for page loads NOT contained in more specific action names. For example, `http://www.site.com/path/page.htm` does not contain actions from `http://www.site.com/path/*`.

Example:<br/><img src={useBaseUrl('img/rum/explore-view-rum.png')} alt="Real User Monitoring" />

## Navigating RUM Dashboards

### RUM Overview Dashboards

> The RUM Overview dashboards show the user experience for performance and requests metrics for your selected application, service, and action. Data is broken down by top geo-locations, operating systems, and browsers. These dashboards include:

Author: ```suggestion
The **RUM Overview**  dashboards (**Application**, **Service**, **Action Type**, and **Action**) show the user experience for performance and requests metrics for your selected application, service, environment, and action. Data is broken down by top geo-locations, operating systems, and browsers. These dashboards include:
```

![RUM Overview](/img/rum/RUM-Overview.png)

![RUM-Overview-Application-Service](/img/rum/RUM-Overview-Application-Service.png)

### RUM TopN Application/Service

The **RUM - TopN - Application** and **RUM - TopN - Application Service** dashboards show top N browsers, operating systems, and geographical locations by load time and requests for the selected **application**, **service**, or **environment**.

Use these dashboards to:
* Find out top N browsers, operating systems, and geolocations by load or requests.
* Understand the slowest and fastest browsers from a rendering perspective or geographical locations from a network perspective.
* Understand XHR and log errors your users are experiencing.
* Find out which browsers and operating systems are in use by your users and where are they are geographically located.

You can select the timing metric type in the **statistic** dropdown on the dashboard header. This will change the browser time metrics types on charts. You can also define the top N number for all charts.

![img](/img/rum/RUM-TopN-Application.png)

### RUM Performance Analytics Application/Service

> The **RUM Performance Analytics** dashboards for **Application**, **Service** and **Environment** show the page performance and requests for a cohort of users specified by selecting the desired combination of dimensions.

Author: ```suggestion
The **RUM Performance Analytics** dashboards for **Application**, **Service**, and **Environment** show the page performance and requests for a cohort of users specified by selecting the desired combination of dimensions.
```

> The **RUM - TopN - Application** and **RUM - TopN - Application Service** dashboards show top N browsers, operating systems, and geographical locations by load time and requests for the selected **application** or **service**.

Reviewer: + RUM Overview - Application Service with Environment

> The **RUM - TopN - Application** and **RUM - TopN - Application Service** dashboards show top N browsers, operating systems, and geographical locations by load time and requests for the selected **application** or **service**.

Author: ```suggestion
The **RUM - TopN - Application** and **RUM - TopN - Application Service** dashboards show top N browsers, operating systems, and geographical locations by load time and requests for the selected **application**, **service**, or **environment**. 
```
